go-apps/meep-rnis/server/rnis.go

```diff
@@ -242,14 +242,33 @@ func Init() (err error) {
 		}
 	}()
 
-	periodicTriggerTicker = time.NewTicker(time.Duration(periodicTriggerIntervals) * time.Second)
+	// Retrieve Meas rep Ue periodic trigger interval from environment variable
+	periodicTriggerInterval := defaultMeasRepUePeriodicTriggerInterval
+	periodicTriggerIntervalEnv := strings.TrimSpace(os.Getenv("MEAS_REP_UE_PERIODIC_TRIGGER_INTERVAL"))
+	if periodicTriggerIntervalEnv != "" {
+		//ignoring last parameter which is the unit, only supporting seconds for now
+		periodicTriggerIntervalVal := periodicTriggerIntervalEnv[:len(periodicTriggerIntervalEnv)-1]
+		periodicTriggerInterval, _ = strconv.Atoi(periodicTriggerIntervalVal)
```

Comment:
Can we use the string time format directly instead of converting to int?

Reply:
using time duration